docs/ui/troubleshooting.md

```diff
@@ -0,0 +1,101 @@
+---
+description: Troubleshooting issues with Prefect Cloud.
+tags:
+    - UI
+    - dashboard
+    - Prefect Cloud
+    - troubleshooting
+---
+
+# Troubleshooting Prefect Cloud
+
+This page provides tips that may be helpful if you run into problems using Prefect Cloud.
+
+## Prefect Cloud and proxies
+
+Proxies intermediate network requests between a server and a client. 
+
+To communicate with Prefect Cloud, the Prefect client library makes HTTPS requests. These requests are made using the [`httpx`](https://www.python-httpx.org/) and [`aiohttp`](https://docs.aiohttp.org/en/stable/) Python libraries. Both libraries respect accepted proxy environment variables, so the Prefect client is able to communicate through proxies. 
```

Comment:
I actually don't think we use aiohttp, checking in https://github.com/PrefectHQ/prefect/pull/7452

Reply:
We've removed aiohttp